### PR TITLE
mapcap_builder_visualizations task #10201

### PR DESCRIPTION
**Pull request by juanignaciosl**
Needed for #10201

@javitonino CR this, please.

### CODE REVIEW

File: lib/tasks/viz_maintenance.rake [hunk 1]
```diff
@@ -151,6 +151,26 @@ namespace :cartodb do
       end
     end
 
+    desc "Have all Builder visualizations mapcapped. Dry mode: `bundle exec rake cartodb:vizs:mapcap_builder_visualizations['--dry']`"
+    task :mapcap_builder_visualizations, [:dry] => :environment do |_, args|
+      dry = args[:dry] == '--dry'
+
+      condition = "version = 3 and type = 'derived'"
+      count = Carto::Visualization.where(condition).count
```
Comment (javitonino) [inline lib/tasks/viz_maintenance.rake]: This is pretty slow (full scan on visualizations table). Maybe it's better to remove it?

**Comment by juanignaciosl** [comment]
@javitonino final CR after your comment, please.

**Comment by juanignaciosl** [comment]
@javitonino I've added `privacy != 'private'` to the visualization search, as private visualizations should not be mapcapped.

@javisantana this will be run so all new ("v3"), non-private maps are mapcapped, and then #10212 will be deployed.

**Comment by javitonino** [comment]
:+1: makes sense to avoid publishing private maps. It won't have any effect, and if we can avoid taking action, we should.

**Comment by juanignaciosl** [comment]
Ignoring tests, as this is just a rake. I'll run it before merging #10212.

**Pull request merged**